ecosystem/indexer-grpc/indexer-grpc-table-info/src/indexer-async-v2/src/lib.rs

```diff
@@ -100,6 +100,21 @@ impl IndexerAsyncV2 {
         self.index_with_annotator(&annotator, first_version, write_sets)
     }
 
+    pub fn handle_pending_on_items(
```

Comment:
What is this function for?

Reply:
we don't expose [TableInfoParser](https://github.com/aptos-labs/aptos-core/blob/d95de84d6c62e68fe8c4abb4133d8492357c4ee0/ecosystem/indexer-grpc/indexer-grpc-table-info/src/indexer-async-v2/src/lib.rs#L201-L206) outside of this indexer async rocksdb, so this fn exposes the logic through the db layer